### PR TITLE
fix: elastic search variable name resolves #50

### LINKED ISSUE #50: SEARCH_SERVER is not being set as expected when Extermal ES
# Description

Aparently with [this change](https://github.com/overhangio/tutor-forum/pull/40/files#diff-3a39d7c4e540ad6bf64e887516aaa76359ea211d5438741db56491b557d4801aL53) the [base env](https://github.com/overhangio/tutor-forum/blob/9b09fdaf3a9714148cf4c240c3c7dd84e539810b/tutorforum/plugin.py#L31) should be changed too. Otherwise the ELASTICSEARCH_HOST is being sent but in the wrong variable:

`Faraday::ConnectionFailed: Failed to open TCP connection to elasticsearch:9200 (getaddrinfo: Name or service not known) (Faraday::ConnectionFailed)`

### PR DESCRIPTION
**Pull request by ghassanmas**

### CODE REVIEW

**Review by DawoudSheraz — APPROVED** [review]

File: tutorforum/plugin.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 }
 
 FORUM_ENV_BASE: dict[str, str] = {
-    "SEARCH_SERVER": "{{ ELASTICSEARCH_SCHEME }}://{{ ELASTICSEARCH_HOST }}:{{ ELASTICSEARCH_PORT }}",
+    "SEARCH_SERVER_ES7": "{{ ELASTICSEARCH_SCHEME }}://{{ ELASTICSEARCH_HOST }}:{{ ELASTICSEARCH_PORT }}",
```
Comment (regisb) [inline tutorforum/plugin.py:31]: I don't understand what is going on here, but I'm almost certain this is not the right fix. This is simply going to crash environment rendering. Why did we set `ENV SEARCH_SERVER_ES7` in the first place?

Comment (MoisesGSalas) [reply tutorforum/plugin.py:31]: From what I gather, [`SEARCH_SERVER_ES7`](https://github.com/openedx/cs_comments_service/blob/d0a8fd7382ee0cbfb9463bbd1003e167d271e768/config/application.yml#L3) is the actual value used by the service and the [entrypoint](https://github.com/overhangio/tutor-forum/blob/58f1cb2e4002c589548a62969662fabe74ec1406/tutorforum/templates/forum/build/forum/bin/docker-entrypoint.sh#L4) was responsible to setting the value to the one defined by `SEARCH_SERVER`.

Comment (regisb) [reply tutorforum/plugin.py:31]: My bad, I thought that the change affected the forum plugin configuration.  LGTM!

**Review by regisb — APPROVED** [review]

**Comment by DawoudSheraz** [comment]
@ghassanmas Hi, this should be good to go. I would advocate releasing a new version when this gets merged because sumac won't have ES config.

**Comment by DawoudSheraz** [comment]
@ghassanmas Hi, catching up on this. Sumac release is near and this PR won't be of use once sumac is out (as ElasticSearch is not going to be there).

**Comment by DawoudSheraz** [comment]
With sumac/v19 release, this is no longer needed as tutor-forum now relies on meilisearch instead of ElasticSearch.

**Pull request closed without merging**